app/src/main/java/org/piramalswasthya/sakhi/ui/login_activity/LoginActivity.kt

```diff
@@ -46,6 +46,8 @@ class LoginActivity : AppCompatActivity() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        // This will block user to cast app screen
+        // Toggle screencast mode for staging & production builds
```

Comment:
What does this mean? Should we change this block when preparing prod/staging builds?

Reply:
This code is for screencasting or screen sharing, it will be blocked in production builds and open in staging builds.